lib/admin-page.php

```diff
@@ -449,6 +449,36 @@ function classicpress_check_can_migrate() {
 	echo "\n</p>\n";
 	echo "</td></tr>\n";
 
+	// Check: Conflicting files
+	$conflicting_files = classicpress_check_conflicting_files();
+	if ( $conflicting_files === false || ! empty( $conflicting_files ) ) {
+		$preflight_checks['conflicting_files'] = false;
+		echo "<tr>\n<td>$icon_preflight_fail</td>\n<td>\n";
+	} else {
+		$preflight_checks['conflicting_files'] = true;
+		echo "<tr>\n<td>$icon_preflight_pass</td>\n<td>\n";
+	}
+	if ( empty( $conflicting_files ) ) {
+		_e(
+			'No conflicting files were found.',
+			'switch-to-classicpress'
+		);
+	} else {
+		echo '<strong class="cp-emphasis">';
+		_e(
+			'Conflicting files detected.',
```

Comment:
a bit more explanation here?  "Conflicting files detected.  These are files that will be <strong>overwritten</strong> by the migration."

Reply:
The logic now is that conflicting files block migration - there's every chance we cannot overwrite them and there's no way to resolve the conflict automatically.

Reply:
OK, that makes sense, the problem is this doesn't explain what "conflicting files" means or how to resolve the issue.  Maybe this then?

> Conflicting files detected. These are files that would be **overwritten** during migration.

Reply:
Works for me.